lab-raziyeh/model/book.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const uuid = require('node-uuid');
+const createError = require('http-errors');
+const storage = require('../lib/storage.js');
+const debug = require('debug')('book:server');
+
+const Book = module.exports = function(title, author, page) {
+  if(!title) throw(createError(400, 'expected Book title'));
+  if(!author) throw(createError(400, 'expected Book author'));
+  if(!page) throw(createError(400, 'expected Book pages'));
+
+  this.id = uuid.v1();
+  this.title = title;
+  this.author = author;
+  this.page = page;
+  this.creationDate = new Date();
```

Comment:
@Raziyehbazargan great model and error catching if an associated property is not provided!